### PR TITLE
Pr@dev@dataset

### PR DESCRIPTION
**Pull request by tjlygdx**
#### Please indicate you've done the following:

- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Made sure commit message follow the rule of [Conventional Commits specification](https://www.conventionalcommits.org/).
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed.

### CODE REVIEW

File: frontend/src/views/system/datasource/dsTable.vue [hunk 1]
```diff
@@ -71,47 +70,64 @@
         </el-col>
       </el-row>
       <el-table
-    :data="dsTableData"
```
Comment (fit2cloud-chenyw) [inline frontend/src/views/system/datasource/dsTable.vue:74]: 代码没有格式化 缩进量没有统一

File: frontend/src/views/system/datasource/dsTable.vue [hunk 2]
```diff
@@ -6,17 +6,16 @@
       </el-col>
       <el-col :span="14" class="right-user">
         <el-input
+          ref="search"
+          v-model="nikeName"
```
Comment (xuwei-fit2cloud) [inline frontend/src/views/system/datasource/dsTable.vue:10]: nikeName -> nickName？

File: frontend/src/views/system/datasource/dsTable.vue [hunk 3]
```diff
@@ -71,47 +70,64 @@
...
+      default: () => {
+      }
+    }
   },
   data() {
     return {
       userDrawer: false,
       dsTableDetail: {},
-      nikeName: "",
+      nikeName: '',
```
Comment (xuwei-fit2cloud) [inline frontend/src/views/system/datasource/dsTable.vue:125]: nickName？下面雷同。

**Review by xuwei-fit2cloud — COMMENTED** [review]
请参考以下标注。

**Review by xuwei-fit2cloud — APPROVED** [review]
LGTM

**Pull request merged**